app_dart/test/src/utilities/mocks.dart

```diff
@@ -27,6 +28,8 @@ class MockTabledataResourceApi extends Mock implements TabledataResourceApi {}
 
 class MockAccessTokenService extends Mock implements AccessTokenService {}
 
+class MockScheduler extends Mock implements Scheduler {}
```

Comment:
not a blocker for this PR, but `mockito` might be a blocker to migrating `app_dart` to null-safety. FYI https://github.com/dart-lang/mockito/blob/master/NULL_SAFETY_README.md

Reply:
Ack, thanks!